### PR TITLE
BaSP-TG-78: Employee worked hours

### PR DESCRIPTION
**Pull request by GonzaloBracamonte**
The hours worked page was created
Form logic to enter hours worked
Validations on the inputs

### CODE REVIEW

**Review by CorinaArreguez — COMMENTED** [review]
Cuando abro work hours me aparece el modal con el mensaje "There was an error".

**Review by Julian25 — APPROVED** [review]

**Review by lucasrendo — APPROVED** [review]

**Review by Evelyn-Heredia — APPROVED** [review]
PR approved though there are some problems to solve during week 15 that don't affect the overall functioning of the app.

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^5.15.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "express": "^4.18.1",
```
Comment (BaltazarCortina1) [inline package.json:10]: Por que express en el front?

File: src/Components/Employee/WorkedHours/index.jsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  const isLoading = useSelector((state) => state.employees.isLoading);
+  const message = useSelector((state) => state.employees.message);
+  const [number, setNumber] = useState({
+    one: '',
+    two: '',
+    three: '',
+    four: '',
+    five: '',
+    six: '',
+    seven: ''
```
Comment (BaltazarCortina1) [inline src/Components/Employee/WorkedHours/index.jsx:27]: Ya que son dias podrian haber tenido el nombre del dia

File: src/Components/Employee/WorkedHours/index.jsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+  });
+
+  const {
+    handleSubmit,
+    register,
+    formState: { errors }
+  } = useForm({
+    mode: 'onBlur',
+    resolver: joiResolver(validationSchema),
+    defaultValues: { one: '', two: '', three: '', four: '', five: '', six: '', seven: '' }
```
Comment (BaltazarCortina1) [inline src/Components/Employee/WorkedHours/index.jsx:48]: Si son numeros el valor inicial deberia ser un numero

File: src/Components/Employee/WorkedHours/index.jsx [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+      <h2>Worked Hours</h2>
+      <div className={styles.topContainer}>
+        <Button>Before</Button>
+        <Input
+          className={styles.dateInput}
+          id={'weekend'}
+          register={register}
+          type="date"
+          error={errors.weekend}
+        />
```
Comment (BaltazarCortina1) [inline src/Components/Employee/WorkedHours/index.jsx:105]: Este input deberia estar dentro de un form

**Pull request merged**